### ISSUE TITLE
WGDashboard - Internal Server Error

### ISSUE THREAD

**Issue by rtripon** [donaldzou/WGDashboard#191]
Hello,
I've noticed that  WGDashboard is failing if I add a second IP address  to main network interface.

Internal Server Error
The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.

Aug 20 07:33:38 ubuntu python3[95386]:   File "/opt/wgdashboard/src/dashboard.py", line 475, in get_conf_status
Aug 20 07:33:38 ubuntu python3[95386]:     ifconfig = dict(ifcfg.interfaces().items())
Aug 20 07:33:38 ubuntu python3[95386]:   File "/usr/local/lib/python3.10/dist-packages/ifcfg/__init__.py", line 70, in interfaces
Aug 20 07:33:38 ubuntu python3[95386]:     return Parser(ifconfig=ifconfig).interfaces
Aug 20 07:33:38 ubuntu python3[95386]:   File "/usr/local/lib/python3.10/dist-packages/ifcfg/parser.py", line 30, in __init__
Aug 20 07:33:38 ubuntu python3[95386]:     self.parse(self.ifconfig_data)
Aug 20 07:33:38 ubuntu python3[95386]:   File "/usr/local/lib/python3.10/dist-packages/ifcfg/parser.py", line 90, in parse
Aug 20 07:33:38 ubuntu python3[95386]:     raise RuntimeError(
Aug 20 07:33:38 ubuntu python3[95386]: RuntimeError: Tried to add broadcast=10.10.90.255 multiple times to ens18, it was already: xxx.xxx.xxx.xxx
Aug 20 07:33:38 ubuntu python3[95386]: 192.168.10.1- - [20/Aug/2022 07:33:38] "GET / HTTP/1.1" 500 -

**Comment by Pashtetickus** [comment]
First, i imported `socket`. Then in `dashboard.py` in `def get_conf_status` i did something like this:

``` python
found = False
for interface in socket.if_nameindex():
    if interface[1] == config_name:
        found = True
return "running" if found else "stopped"
```

insted of 
``` python
ifconfig = dict(ifcfg.interfaces().items())
return "running" if config_name in ifconfig.keys() else "stopped"
```

I also faced a problem with the endpoint and solved it with this #205

**Comment by buddyqc69** [comment]
i have fix this with this comments : https://github.com/donaldzou/WGDashboard/issues/225#issuecomment-1606067170

But now the error 500 is my last probleme for the moment, do you have fix for this ?
